url78eec0e0b6_Orikuro_vic_tgz-pJ8-net_nexon_vindictus_itemcomparer_inport_StaticImportJ8/src/net/nexon/vindictus/itemcomparer/inport/StaticImport.java

```diff
@@ -0,0 +1,2125 @@
+		boni.add(new ItemSetBonus(180, 0, 84));
+		boni.add(new ItemSetBonus(240, 0, 125));
+
+		ItemSet iset = new ItemSet(setname, boni);
+		iset.setUsefulChars(arisha);
+
+		Gloves g = new Gloves(iset, "Gloves", 60, ItemTyp.HEAVY, 335, 13, 86,
+				20, 10);
+		Shoes s = new Shoes(iset, "Shoes", 60, ItemTyp.HEAVY, 322, 12, 86, 19,
+				10);
+		Pants p = new Pants(iset, "Pants", 60, ItemTyp.LIGHT, 512, 16, 98, 23,
+				11);
+		Helm h = new Helm(iset, "Helm", 60, ItemTyp.HEAVY, 452, 15, 86, 22, 12);
+		Armor a = new Armor(iset, "Armor", 60, ItemTyp.HEAVY, 577, 15, 94, 21,
+				13);
+		g.setStamina(2);
+		s.setStamina(2);
+		p.setStamina(3);
+		h.setStamina(2);
+		a.setStamina(3);
+
+		shoes.add(s);
+		gloves.add(g);
+		pants.add(p);
+		armors.add(a);
+		helms.add(h);
+	}
+
+	static void generateStriker() {
+		String setname = "L Strike";
+		sets.add(setname);
+		List<ItemSetBonus> boni = new ArrayList<ItemSetBonus>();
+		boni.add(new ItemSetBonus(150, 0, 12));
+		boni.add(new ItemSetBonus(165, 0, 24));
+		boni.add(new ItemSetBonus(180, 0, 39));
+		boni.add(new ItemSetBonus(210, 0, 52));
+
+		ItemSet iset = new ItemSet(setname, boni);
+		iset.setUsefulChars(arisha);
+
+		Gloves g = new Gloves(iset, "Gloves", 54, ItemTyp.CLOTH, 256, 0, 98,
+				18, 8);
+		Shoes s = new Shoes(iset, "Shoes", 54, ItemTyp.CLOTH, 267, 0, 99, 17, 8);
+		Pants p = new Pants(iset, "Pants", 54, ItemTyp.CLOTH, 300, 0, 97, 19, 9);
+		Helm h = new Helm(iset, "Helm", 54, ItemTyp.CLOTH, 272, 0, 98, 15, 8);
+		Armor a = new Armor(iset, "Armor", 54, ItemTyp.CLOTH, 326, 0, 99, 17,
+				11);
+
+		shoes.add(s);
+		gloves.add(g);
+		pants.add(p);
+		armors.add(a);
+		helms.add(h);
+	}
+
+	static void generateRegina_3_min() {
+		String setname = "Regina (Orange minimum)";
+		sets.add(setname);
+		List<ItemSetBonus> boni = new ArrayList<ItemSetBonus>();
+		boni.add(new ItemSetBonus(0, 0, 0));
+		ItemSetBonus sonderbonus1 = new ItemSetBonus(165, 60, 81);
+		sonderbonus1.setSta(5);
+		boni.add(sonderbonus1);
+
+		ItemSetBonus sonderbonus2 = new ItemSetBonus(220, 80, 108);
+		sonderbonus2.setSta(6);
+		boni.add(sonderbonus2);
+		
+		ItemSetBonus sonderbonus3 = new ItemSetBonus(275, 100,135);
+		sonderbonus3.setSta(7);
+		boni.add(sonderbonus3);
+		
+		ItemSetBonus sonderbonus4 = new ItemSetBonus(330, 120, 162);
+		sonderbonus4.setSta(10);
+		boni.add(sonderbonus4);
+
+		ItemSet iset = new ItemSet(setname, boni);
+		iset.setUsefulChars(char_melee + " " + magic);
+
+		Gloves g = new Gloves(iset, "Gloves", 90, ItemTyp.PLATE, 1255, 144,
+				194, 72, 13);
+		Shoes s = new Shoes(iset, "Shoes", 90, ItemTyp.PLATE, 1255, 144, 194,
+				72, 13);
+		Helm h = new Helm(iset, "Helm", 90, ItemTyp.PLATE, 1255, 144, 194, 72,
+				13);
+
+		Pants p = new Pants(iset, "Pants", 90, ItemTyp.PLATE, 1355, 144, 194,
+				72, 18);
+		Armor a = new Armor(iset, "Armor", 90, ItemTyp.PLATE, 1422, 144, 194,
+				72, 16);
+
+		shoes.add(s);
+		gloves.add(g);
+		pants.add(p);
+		armors.add(a);
+		helms.add(h);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 1695:1734, 1737:1776, 1780:1819, 1823:1862.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.